src/handlers/submitHomework.ts

```diff
@@ -0,0 +1,81 @@
+// POST
+import { CustomDynamoDB } from "../dynamodb/database";
+import responseHelper from "../helpers/responseHelper";
+import { validateToken } from "../helpers/validations";
+import { v4 as uuidv4 } from 'uuid';
+
+const userDB = new CustomDynamoDB('local-bootcamp-classroom-api-users', 'email')
+const homeworkDB = new CustomDynamoDB('local-bootcamp-classroom-api-homeworks', 'id')
+const homeworkSubmissionDB = new CustomDynamoDB('local-bootcamp-classroom-api-homeworkSubmissions', 'id')
+
+export const handler = async(event: any) => {
+    const tokenData = await validateToken(event.headers.Authorization)
+
+    if(!tokenData){
+        return responseHelper(403, 'Fail Authenticating');
+    }
+ 
+    const body = JSON.parse(event.body);
+
+    const {
+        studentId, 
+        homeworkId,
+        submission,
+    } = body;
+    
+    if(!studentId || studentId.trim() === '') {
+            return responseHelper(400, 'Student ID not provided')
+    }
+
+    if(tokenData.userEmail !== studentId){
+        return responseHelper(403, 'Student email not in token')
```

Comment:
Instead of `not in token` I think its a better description `doesn't match token`